packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -592,8 +594,12 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
 
     return (
       <div
-        ref={ (el: HTMLDivElement) =>
-          this._splitButtonContainers.set(item.key, el)
+        ref={ (el: HTMLDivElement) => {
+          this._splitButtonContainers.set(item.key, el);
+          if (el && 'onpointerdown' in el) {
+            this._events.on(el, 'pointerdown', this._onPointerDown, true);
```

Comment:
This shouldn't be here. The ref will be called whenever anything is mounted so it will attempt to add another event listener. We should consider moving the split button into a separate class/stateless component since it seems like it should have its own state management.

Reply:
Agreed. This should be in `componentDidMount`.

Reply:
We should definitely use a separate component. But, if it's possible, it would be great if contextualmenu could use the existing split button, rather than a split button that is specific to contextualmenu.